Files/Views/SettingsPages/Experimental.xaml

```diff
@@ -45,7 +46,20 @@
                     Header="Cache files and folders for better performance"
                     HeaderTemplate="{StaticResource CustomHeaderStyle}"
                     IsOn="{Binding UseFileListCache, Mode=TwoWay}" />
+
+                <controls:NumberBox 
+                    x:Uid="PreemptiveCacheParallelLimits"
+                    HorizontalAlignment="Left"
+                    Header="Preemptive cache parallel limit (0 turns off the preemptive cache)" 
+                    SpinButtonPlacementMode="Inline"
+                    Width="450"
+                    Value="{Binding PreemptiveCacheParallelLimit, Mode=TwoWay}" 
+                    HeaderTemplate="{StaticResource CustomHeaderStyle}"
+                    IsEnabled="{Binding UseFileListCache, Mode=OneWay}"
+                    Minimum="0" 
+                    HeaderTemplate="{StaticResource CustomHeaderStyle}"
```

Comment:
@yaichenbaum I already did that change, so we have a duplicates here

Reply:
Whoops, sorry about that!